content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -126,79 +126,100 @@ Registration
+- :guilabel:`Phone`: phone number including the country code (e.g., `+32` in Belgium).
+- :guilabel:`Email`: this the email Odoo can contact you to regarding your Peppol registration.
 
-   .. image:: electronic_invoicing/peppol-demo-mode.png
-      :alt: Peppol demo mode selection
+If you want to explore or demo Peppol, you can choose to register in :guilabel:`Demo` mode.
+Otherwise, select :guilabel:`Live`.
 
-.. note::
-   When testing Peppol, the system parameter `account_peppol.edi.mode` can be changed to `test`.
-   Then, a radio button appears with the option to register on the test server.
+.. tip::
+   - Selecting :guilabel:`Demo` simulates everything in Odoo. There is no sending, receiving, or
+     partner verification.
+   - For **advanced users only**, it is possible to run tests on Peppol's test network. The server
+     allows to register on Peppol, send, and receive test invoices to and from other
+     participants. To do so, enable the :doc:`../../../general/developer_mode`, open the
+     **Settings** app, go to :menuselection:`Technical --> System Parameters`, and search for
+     `account_peppol.edi.mode`. Open the parameter and change the :guilabel:`Value` to `test`. Go
+     back to the Peppol setup menu in the **Settings** app. The option :guilabel:`Test` is now
+     available.
 
    .. image:: electronic_invoicing/peppol-system-parameter.png
       :alt: Peppol test mode parameter
 
-   .. image:: electronic_invoicing/peppol-test-mode-settings.png
-      :alt: Peppol test mode selection
-
-Now, you can request a verification code to be sent to you by clicking :guilabel:`Verify phone
-number`.
-
-.. image:: electronic_invoicing/peppol-registration-verify.png
-   :alt: phone validation request verification
+.. seealso::
+   - `Peppol EAS - European Commision <https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/Code+lists/>`_
+   - `Peppol Endpoint - OpenPeppol eDEC Code Lists <https://docs.peppol.eu/edelivery/codelists/>`_
+     (open the "Participant Identifier Schemes" as HTML page)
 
-A text message containing a code is sent to the phone number provided to finalize the verification
-process.
+When set up, request a verification code to be sent to you by clicking
+:guilabel:`Send a registration code by SMS`. A text message containing a code is sent to the phone
+number provided to finalize the verification process.
 
-.. image:: electronic_invoicing/phone-registration.png
+.. image:: electronic_invoicing/peppol-phone-verification.png
    :alt: phone validation
 
-Once you enter the code and click :guilabel:`Confirm`, you can see that your registration is pending
-activation. From this point onwards, the default journal for receiving Vendor Bills can be set.
-
-.. image:: electronic_invoicing/peppol-registration-pending.png
-   :alt: pending application
+Once you enter the code and click :guilabel:`Register`, your Peppol participant status is updated.
```

Comment:
not sure about that, because you don't really click *in order to* update the status :thinking: the status is updated because things are happening behind the scenes, but registering is technically just about registering on our access point